### DISCUSSION TITLE
Dynamic Safety Stock

### DISCUSSION THREAD

**Discussion by JATDALTON** [frePPLe/frepple#664] [Q&A] [answered]
Documentation talks about using a dynamic safety stock by using the min/max calendar fields.

Is this as simple as creating a calendar with a default value of 7 if I want 7 days of safety stock? 

Or does it work by assigning different safety stock values based on the date, i.e. the value of the calendar needs to be the safety stock level and appropriate date range, not the number of days I want?

**Comment by jdetaeye** [comment, marked as answer]
>>Is this as simple as creating a calendar with a default value of 7 if I want 7 days of safety stock?

You set the buffer.minimum field if you want a static safety stock quantity over the complete horizon.
You set the buffer.minimum_calendar field if you want the safety stock quantity to change over time:   10 this month, 15 next month,  20 in 2 months, etc...

But it remains a quantity.

The inventory planning module (only available in the enterprise and cloud editions) provides functionality to define safety stock is much more detail:  expressed as days of demand, smart calculation based on variability of demand and supply, easy mass management of inventory policies, smart calculation of replenishment quantities, etc...

**Reply by JATDALTON** [reply]
Appreciate the response. This was what I was seeing, just wanted to make sure I wasn't missing something. Cloud edition is on the horizon I believe.
